### PR TITLE
New term entry for Java Math.nextUp().

### LINKED ISSUE #908: [Term Entry] Java Math Methods: `.nextUp()`
### Reason/inspiration (optional)

Under `java/concepts/math-methods/terms/`, create a new term entry for the `Math.nextUp()` Java method.

- In the `terms/` folder, make a new folder called `next-up`
- Inside the `next-up` folder, create a new markdown file called `next-up.md`.
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Also make sure to include an "Example" section with a \```java code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by shivanshugour45**
### Description

Creating next-up.md

Closes #908 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/math-methods/terms/next-up/next-up.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Math.nextUp(float val)
+```
+
+or
+
+```pseudo
+Math.nextUp(double val)
+```
+
+Following are some of the special cases for this method-
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/next-up/next-up.md:30]:
```suggestion
The following are some of the special cases for this method:
```

File: content/java/concepts/math-methods/terms/next-up/next-up.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.nextUp()`** method returns the floating-point value adjacent to the argument passed as the parameter in the direction of positive infinity. Argument passed can be either of type `double` or `float`.
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/next-up/next-up.md:16]: Move "Argument passed can be either of type `double` or `float`." to the "Syntax" section, then you can have one ```pseudo block reading `Math.nextUp(val)`.

File: content/java/concepts/math-methods/terms/next-up/next-up.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Syntax
+
+```pseudo
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/next-up/next-up.md:20]: See my prior comment about combining these into one ```pseudo block.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@shivanshugour45 I did a first review and left a few comments.

**Comment by shivanshugour45** [comment]
@SSwiniarski, Suggested changes are done.

**Review by HishamT — APPROVED** [review]
LGTM!

File: content/java/concepts/math-methods/terms/next-up/next-up.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.nextUp()`** method returns the floating-point value adjacent to the argument passed as the parameter in the direction of positive infinity.
+
+## Syntax
+
+#### Note: Argument passed can be either of type `double` or `float`.
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/next-up/next-up.md:20]:
```suggestion
Argument passed can be either of type `double` or `float`.
```

I'd also move this line after the ```pseudo block.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@shivanshugour45 I have one more comment.

**Comment by shivanshugour45** [comment]
Suggested updates done.

**Review by SSwiniarski — APPROVED** [review]
@shivanshugour45 Looks good.

**Comment by yangc95** [comment]
@shivanshugour45 Live link: https://www.codecademy.com/resources/docs/java/math-methods/nextUp
(I made some changes prior to merge here: [87cf532](https://github.com/Codecademy/docs/pull/1142/commits/87cf532575406626d0e372cf8d926014ce2db3d5)). I changed the intro paragraph a bit to stray slightly away from existing documentation on this method.

Screen Shot 2022-10-21 at 10 29 43 PM
You're currently anonymous but if you wish to become public, make sure to add your GitHub to your Codecademy profile.

cc: @HishamT @SSwiniarski

**Pull request merged**